### DISCUSSION TITLE
Partial Cash On Delivery (COD) - Pay 10% now & rest upon delivery. - Reduce Fake Orders

### DISCUSSION THREAD

**Discussion by Rana-Biswas** [inspireui/support#5772] [New Feature/Ideas Requests]
We have been using Fluxstore WooCommerce for some time now and have been extremely pleased with its performance and the features it offers. However, we believe that there is an exciting opportunity for improvement that could greatly benefit both our business and many other users like us.

We kindly request that you consider implementing a new feature: "Partial Cash On Delivery (COD) - Pay 10% now & rest upon delivery." This feature would be a game-changer for our business and many others who rely on Cash On Delivery as a payment method.

We have identified a free plugin called "**WooCommerce Smart COD**" (https://es-co.wordpress.org/plugins/wc-smart-cod/) **and its pro version** (https://woosmartcod.com/product/woocommerce-smart-cod-pro/), which offers the partial payment feature with existing payment gateways. The pro version specifically supports partial payments, allowing customers to pay a portion of their order upfront, typically 30%, and the remaining amount upon delivery.

Here's why we believe this feature would be a significant enhancement to Fluxstore WooCommerce:

1. **Enhanced Payment Flexibility:** Many customers prefer the option to make a partial payment when placing an order. This feature would cater to a wider audience and potentially increase conversion rates.
2. **Reduce Fake Orders & Risk:** Partial COD payments will reduce fake orders and the risk associated with large COD orders, ensuring that a significant portion of the payment is received upfront.
4. **Competitive Advantage:** Implementing this feature would give Fluxstore WooCommerce an edge over other mobile app solutions, attracting more businesses looking for a versatile e-commerce app.
6. **Integration with mStore API Template:** We request that this feature be implemented in such a way that it seamlessly integrates with the mStore API template, ensuring a great user experience for all web view payment methods. https://ibb.co/LSBtTw6

We understand that implementing new features requires time and resources, but we believe that this addition would greatly benefit Fluxstore WooCommerce users.

@minhcasi @HanielQC @luyx2412 @vumanhsn97

**Comment by Rana-Biswas** [comment]
Please vote for this feature to avoid the hassle of Fake Orders

**Reply by Jossua-demavie** [reply]
I believe this is now possible but with the WooCommerce Deposits plugin.

I now believe they need to integrate it into Fluxstore Manager to allow vendors to set this up without having to go to the website each time.
